public/js/safetymaps/modules/vrgz/route.js

```diff
@@ -0,0 +1,131 @@
+/*
+ *  Copyright (c) 2020 B3Partners (info@b3partners.nl)
+ *
+ *  This file is part of safetymapDBK
+ *
+ *  safetymapDBK is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  safetymapDBK is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with safetymapDBK. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+var dbkjs = dbkjs || {};
+window.dbkjs = dbkjs;
+dbkjs.modules = dbkjs.modules || {};
+
+dbkjs.modules.route = {
+    id: "dbkjs.modules.route",
+    options: null,
+    reloadTimeOut: null,
+    routeUrlSimacar: "/simacar_export/Route.txt",
+    routeUrlNotResponded: false,  
+    register: function () {
+        var me = this;
+        var pointArray;
+        var routeLayer;
+
+        me.options = $.extend({ 
+            provider: "simacar",
+            routeColor: "blue",
+            reloadInterval: 5000
+        }, me.options);
+        
+        window.clearTimeout(me.reloadTimeOut);   
+        
+        me.getRoute();
+    },
+    getRoute: function(reload) {
+        var me = this;
+
+        if (me.options.provider === "simacar")            
+            me.getSimacarRoutesAsPointArray(function (pointArray) {
+                me.addRouteLayerToMap(me.createRouteLayer(pointArray), reload);
```

Comment:
Don't add layer everytime you get a new route, just do it once on startup and on update remove all features and only do `routeLayer.layer.addFeatures()`